backend/urls.py

```diff
@@ -18,12 +18,23 @@
 from django.contrib import admin
 from django.urls import include, path
 from lti_tool.views import jwks, OIDCLoginInitView
-from backend.ccm.canvas_api.course_api_handler import CanvasCourseAPIHandler
 from backend.ccm.lti_config import CCMLTILaunchView
+from django.http import HttpResponseForbidden
 import watchman.views
 
 from backend import views
 
+def disabled_admin_login(request, *args, **kwargs):
+    return HttpResponseForbidden("View is disabled.")
+
+# Disable the Django admin login page by assigning the simple view directly.
+if not getattr(settings, "ENABLE_BACKEND", False):
+    # In normal deployments, disable the admin login page so the app doesn't expose admin url.
+    # For local load testing where you need the backend enabled set ENABLE_BACKEND=True
+    admin.site.login = disabled_admin_login
```

Comment:
By Default, Admin ( with super_user and is_staff) role are given access to `/admin` page. it's Django checking pattern. We don't control unless we disable `path('admin/', admin.site.urls),`